requirements/dev.txt

```diff
@@ -2,7 +2,7 @@ coverage>=4.4.0
 flake8>=3.3.0
 tox>=2.2.1
 pytest-cov>=2.4.0
-pylint>=1.7.2
+pylint>=1.7.2,<2.0
```

Comment:
Any particular reason why this was necessary?

Reply:
pip was trying to install a post 2.0 version which required py3.

Reply:
Ahh ok, thanks